google/cloud/compute/CMakeLists.txt

```diff
@@ -223,6 +223,23 @@ foreach (dir IN LISTS service_dirs)
     list(APPEND compute_lib_targets "google_cloud_cpp_compute_${short_dir}")
 endforeach ()
 
+add_library(google_cloud_cpp_compute INTERFACE)
+target_link_libraries(
+    google_cloud_cpp_compute
+    PUBLIC
+    INTERFACE ${compute_lib_targets})
+set_target_properties(
+    google_cloud_cpp_compute
+    PROPERTIES EXPORT_NAME "google-cloud-cpp::compute"
+               VERSION ${PROJECT_VERSION}
+               SOVERSION ${PROJECT_VERSION_MAJOR})
+target_include_directories(
+    google_cloud_cpp_compute
+    INTERFACE $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
+              $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}>
+              $<INSTALL_INTERFACE:include>)
```

Comment:
Are you sure this is needed?  They should be inherited with the  `target_link_libraries( ... PUBLIC ...)`  command.  OTOH, they don't hurt either.